src/main/java/racingcar/global/util/Validator.java

```diff
@@ -0,0 +1,44 @@
+package racingcar.global.util;
+
+import racingcar.global.enums.ErrorMessage;
+
+public class Validator {
+    private static Validator instance;
+
+    public static Validator getInstance() {
+        if (instance == null) {
+            instance = new Validator();
+        }
+        return instance;
+    }
+
+    public void validateName(String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException(
+                    ErrorMessage.INVALID_NAME_LENGTH.getMessage());
+        }
+        if (name.isBlank()) {
+            throw new IllegalArgumentException(
+                    ErrorMessage.NOT_ALLOWED_EMPTY_SPACE.getMessage());
+        }
+    }
+
+    public Integer validateAndReturnCount(String count) {
+        if (count.isBlank()) {
+            throw new IllegalArgumentException(ErrorMessage.NOT_ALLOWED_EMPTY_SPACE.getMessage());
+        }
+
+        try {
+            return Integer.parseInt(count);
+        } catch (Exception e) {
+            throw new IllegalArgumentException(ErrorMessage.WRONG_INPUT.getMessage());
+        }
+    }
+
+    public Integer isPositiveNumber(Integer count) {
+        if (count > 0) {
+            return count;
+        }
+        throw new IllegalArgumentException(ErrorMessage.ALLOWED_ONLY_POSITIVE.getMessage());
+    }
```

Comment:
메서드 네이밍을 봤을 때 예외를 던지는 건 _validateAndReturnCount()_ 에 오는 게 더 자연스러울 것 같습니다 !

`is`로 시작하는 메서드 이름도 보통 `boolean` 타입을 반환할 때 사용하는 네이밍 컨벤션이기도 하구요.

시도횟수 입력이 빈 문자열 또는 whitespace 일 경우는 정수형으로 변환하는 과정에서 캐치할 수 있을 것 같아요. 

```suggestion
    public Integer validateAndReturnCount(String input) {
        try {
            Integer count = Integer.parseInt(count);
            
            if (isNotPositiveNumber(count)) {
                throw new IllegalArgumentException(ErrorMessage.ALLOWED_ONLY_POSITIVE.getMessage());
            }
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException(ErrorMessage.WRONG_INPUT.getMessage());
        }
    }

    public boolean isNotPositiveNumber(Integer count) {
        return count <= ZERO;
    }
```

Reply:
앗 그렇군요! 네이밍 컨벤션에 대해서도 더 공부해봐야겠네요!👍👍